docs/smart_scripts.md

```diff
@@ -4372,6 +4371,58 @@ This is the probability of the event to occur as a percentage from 0-100. So, if
 </p></td>
 <td><p>Farthest unit on the threat list</p></td>
 </tr>
+<tr class="odd">
+<td><p>SMART_TARGET_PLAYER_WITH_AURA</p></td>
+<td><p>201</p></td>
+<td><p>spellID</p></td>
+<td><p>Negative (0/1)</p></td>
+<td><p>MaxDist</p></td>
+<td><p>MinDist</p></td>
+<td><p><br />
+</p></td>
+<td><p><br />
+</p></td>
+<td><p><br />
+</p></td>
+<td><p><p>Number to resize the target list</p></td>
+<td><p><p>Target players with aura</p></td>
+</tr>
+<tr class="even">
+<td><p>SMART_TARGET_RANDOM_POINT</p></td>
```

Comment:
This Target type works only for SMART_ACTION_SUMMON_CREATURE, SMART_ACTION_MOVE_TO_POS and SMART_ACTION_JUMP_TO_POS (Yet)